parmed/rdkit/__init__.py

```diff
@@ -0,0 +1,8 @@
+"""
+A package dealing with RDKit integration.
+"""
+
+from parmed.rdkit.rdkit import RDKit
+
+load_rdkit = RDKit.load
+from_smile = RDKit.from_smile
```

Comment:
- ah, typos (always think about a 'smile' :D) 

> Does RDKit really use from_smile?

No, It does not have this method. I want to add since I feel frustrated with its (RDKit) syntax (this is understandable RDKit is written in C++ with different interfaces).